noetic/distribution.yaml

```diff
@@ -265,6 +265,13 @@ repositories:
       url: https://github.com/ros/eigen_stl_containers.git
       version: master
     status: maintained
+  executive_smach:
+    source:
+      test_pull_requests: true
+      type: git
+      url: https://github.com/ros/executive_smach.git
+      version: noetic-devel
```

Comment:
It looks like the branch is called "noetic_devel" on the source repository (which is why this failed CI).  Personally, I'd suggest renaming the source branch to "noetic-devel", as that is a lot more idiomatic.

Reply:
updated